client/state/invites/selectors.js

```diff
@@ -27,3 +27,20 @@ export function getInvitesForSite( state, siteId ) {
 	}
 	return invites;
 }
+
+/**
+ * Returns true if currently requesting an invite resend for the given site, or false
+ * otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @param  {String}  inviteId Invite ID
+ * @return {Boolean}        Whether invites are being requested
+ */
+export function isRequestingInviteResend( state, siteId, inviteId ) {
+	const siteResendRequests = state.invites.requestingInviteResend[ siteId ];
+	if ( ! siteResendRequests ) {
+		return null;
+	}
+	return !! siteResendRequests[ inviteId ];
```

Comment:
I think you could also use `return get( state, [ 'invites', 'requestingInviteResend', siteId, inviteId ], false );`